BookingsApi/BookingsApi/Validations/ParticipantRequestValidation.cs

```diff
@@ -22,6 +23,7 @@ public ParticipantRequestValidation()
             RuleFor(x => x.DisplayName).NotEmpty().WithMessage(NoDisplayNameErrorMessage);
             RuleFor(x => x.CaseRoleName).NotEmpty().WithMessage(NoCaseRoleNameErrorMessage);
             RuleFor(x => x.HearingRoleName).NotEmpty().WithMessage(NoHearingRoleNameErrorMessage);
+            RuleFor(x => x.TelephoneNumber).NotEmpty().WithMessage(NoTelephoneErrorMessage).When(x => x.HearingRoleName == "Staff Member");
```

Comment:
could we declare the staff member as a string const somewhere, then we can reference it here and also in the test instead of using magic strings?

Reply:
done